src/SFA.DAS.AANHub.Data/Repositories/MemberLeavingReasonsWriteRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.AANHub.Data.Repositories;
+
+[ExcludeFromCodeCoverage]
+public class MemberLeavingReasonsWriteRepository : IMemberLeavingReasonsWriteRepository
+{
+    private readonly AanDataContext _aanDataContext;
+
+    public MemberLeavingReasonsWriteRepository(AanDataContext aanDataContext) => _aanDataContext = aanDataContext;
+
+    public void Create(MemberLeavingReason memberLeavingReason) => _aanDataContext.MemberLeavingReasons.Add(memberLeavingReason);
```

Comment:
You can take a enumerable and use AddRange here instead of adding one at a time.

Reply:
ok